common/src/main/java/me/lucko/luckperms/common/locale/Message.java

```diff
@@ -90,6 +92,9 @@
  */
 public interface Message {
 
+    static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd '@' HH:mm:ss")
```

Comment:
Wondering if this should be translatable/configurable? Many countries have vastly different time formatting standards

Reply:
Reason I did it in that format is that’s what ‘ VerboseListener’ and ‘ExportCommand’ use for date format. But it would be cool to add 😅

Reply:
Ah okay, no worries in that case then I think!